### PR TITLE
Spacemouse debugging

### PR DESCRIPTION
**Pull request by gautams3**
Add a test file to debug spacemouse output

### CODE REVIEW

File: serl_robot_infra/franka_env/spacemouse/spacemouse_test.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+""" Test the spacemouse output. """
+
+import time
+import numpy as np
+from serl_robot_infra.franka_env.spacemouse.spacemouse_expert import SpaceMouseExpert
+np.set_printoptions(precision=3, suppress=True)
```
Comment (jianlanluo) [inline serl_robot_infra/franka_env/spacemouse/spacemouse_test.py:6]: can you move this to test_spacemouse() ? this should be only locally valid

File: serl_robot_infra/franka_env/spacemouse/spacemouse_test.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+def test_spacemouse():
+    """ Test the SpaceMouseExpert class. """
```
Comment (jianlanluo) [inline serl_robot_infra/franka_env/spacemouse/spacemouse_test.py:9]: can you add detailed descriptions?

**Review by jianlanluo — CHANGES_REQUESTED** [review]

**Comment by jianlanluo** [comment]
also can you fix the pre-commit failing ?

**Comment by gautams3** [comment]
1. addressed comments by @jianlanluo 
2. fixed pre-commit failures

These are a lot of commits for a minor change. It might help to squash-and-merge instead of the default.

File: serl_robot_infra/franka_env/spacemouse/spacemouse_test.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+def test_spacemouse():
+    """Test the SpaceMouseExpert class.
+
+    This interactive test prints the action and buttons of the spacemouse at a rate of 10Hz.
+    The user is expected to move the spacemouse and press its buttons while the test is running.
+    It keeps running until the user stops it.
+
+    """
+    import time
+    import numpy as np
```
Comment (jianlanluo) [inline serl_robot_infra/franka_env/spacemouse/spacemouse_test.py:13]: import should be on top of the file

**Review by jianlanluo — CHANGES_REQUESTED** [review]

File: serl_robot_infra/franka_env/spacemouse/spacemouse_test.py (hunk 3 above)
Comment (gautams3) [reply serl_robot_infra/franka_env/spacemouse/spacemouse_test.py:13]: I think I misunderstood then. You want imports to be on top, but the setting of `numpy` precision options to be in the test? both of these are global changes.

Comment (jianlanluo) [reply serl_robot_infra/franka_env/spacemouse/spacemouse_test.py:13]: import should always go to the top of a py file,

np print precesion is local to your function, e.g., you can change in another function; so it should go to the function which actually needs it

Comment (gautams3) [reply serl_robot_infra/franka_env/spacemouse/spacemouse_test.py]: np print precision is not local. But we can make it local with a context manager. just pushed a change

**Review by jianlanluo — APPROVED** [review]

**Pull request merged**